environment.yml

```diff
@@ -17,6 +17,8 @@
    - pip
    - ffmpeg
    - pytest
+   - ipywidgets
    - pip:
      - python-awips
      - mpldatacursor
+     - drilsdown
```

Comment:
Brian, did you mean to push this? Because these don't look like changes we should take. If it was a mistake, we can handle.